service/framework.go

```diff
@@ -422,9 +459,10 @@ func NewVersionedResources(config Config, k8sClient kubernetes.Interface, awsCon
 		// Clusters without a version use the legacy resource.
 		"":      legacyResources,
 		"0.1.0": legacyResources,
-		"0.2.0": resources,
+		"0.2.0": resourcesV1,
 		"1.0.0": legacyResources,
-		"2.0.0": resources,
+		"2.0.0": resourcesV1,
```

Comment:
Nickpick but `resourcesV1` mapping to `2.0.0` is confusing for me. Could we use `resourcesV2` and `resourcesV2_0_1`?

Reply:
Sure, much better, done.